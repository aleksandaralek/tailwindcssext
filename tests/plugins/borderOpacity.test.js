import { quickPluginTest } from '../util/run'

quickPluginTest('borderOpacity', {
  safelist: [
    // Arbitrary values
    'border-opacity-[12%]',
    'border-opacity-[var(--my-value)]',
  ],
}).toMatchSnapshot()
