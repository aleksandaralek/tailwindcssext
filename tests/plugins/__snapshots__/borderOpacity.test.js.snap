// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the 'borderOpacity' plugin 1`] = `
"
.border-opacity-0 {
  --tw-border-opacity: 0;
}

.border-opacity-10 {
  --tw-border-opacity: .1;
}

.border-opacity-100 {
  --tw-border-opacity: 1;
}

.border-opacity-15 {
  --tw-border-opacity: .15;
}

.border-opacity-20 {
  --tw-border-opacity: .2;
}

.border-opacity-25 {
  --tw-border-opacity: .25;
}

.border-opacity-30 {
  --tw-border-opacity: .3;
}

.border-opacity-35 {
  --tw-border-opacity: .35;
}

.border-opacity-40 {
  --tw-border-opacity: .4;
}

.border-opacity-45 {
  --tw-border-opacity: .45;
}

.border-opacity-5 {
  --tw-border-opacity: .05;
}

.border-opacity-50 {
  --tw-border-opacity: .5;
}

.border-opacity-55 {
  --tw-border-opacity: .55;
}

.border-opacity-60 {
  --tw-border-opacity: .6;
}

.border-opacity-65 {
  --tw-border-opacity: .65;
}

.border-opacity-70 {
  --tw-border-opacity: .7;
}

.border-opacity-75 {
  --tw-border-opacity: .75;
}

.border-opacity-80 {
  --tw-border-opacity: .8;
}

.border-opacity-85 {
  --tw-border-opacity: .85;
}

.border-opacity-90 {
  --tw-border-opacity: .9;
}

.border-opacity-95 {
  --tw-border-opacity: .95;
}

.border-opacity-\\[12\\%\\] {
  --tw-border-opacity: 12%;
}

.border-opacity-\\[var\\(--my-value\\)\\] {
  --tw-border-opacity: var(--my-value);
}
"
`;
